### DISCUSSION TITLE
What does the number after repositories in my profile mean?

### DISCUSSION THREAD

**Discussion by Thecorator** [community/community#23130] [New to GitHub]
<p>I only have three repositories but the number after <strong>Repositories</strong> is 5 and would like to know what this number means.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Most likely a caching issue: :sweat_smile:</p><aside class="quote quote-modified" data-post="2" data-topic="256751">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/display-error-maybe-my-account-shows-wrong-number-of-repositories/256751/2">Display Error maybe. My account shows wrong number of repositories.</a> <a class="badge-wrapper  bullet" href="/c/github-help/48"><span class="badge-category-bg" style="background-color: #6a737d;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Ask questions about your GitHub account and the core features of GitHub.">GitHub Help</span></a>
  </div>
  <blockquote>
    I’ve seen a few reports like this over the last few days, so I suspect there’s some cache that’s slow to update. In the thread linked below someone mentioned the problem went away after they created a new repository. sweat_smile 


So I guess either do that, or send a bug report to <a href="https://support.github.com/request" class="inline-onebox">GitHub Support</a> so they can get on a proper fix. slightly_smiling_face
  </blockquote>
</aside>
